csrankings-c.csv

```diff
@@ -579,6 +579,7 @@ Chris Bailey 0002,University of York,https://www-users.cs.york.ac.uk/~chrisb,QQ_
 Chris Bailey-Kellogg,Dartmouth College,http://www.cs.dartmouth.edu/~cbk,NOSCHOLARPAGE
 Chris Bartone,Ohio University,https://www.ohio.edu/engineering/about/people/profiles.cfm?profile=bartone,NOSCHOLARPAGE
 Chris Biemann,University of Hamburg,https://www.inf.uni-hamburg.de/en/inst/ab/lt/people/chris-biemann,BdwP-3QAAAAJ
+Chris Brown,Virginia Tech,https://website.cs.vt.edu/people/faculty/chris-brown.html,MmgiKNkAAAAJ

```

Comment:
Please send a note to dblp@dblp.org asking them to create a disambiguation entry for this faculty member.